### PR TITLE
Update GetOsInfo for Windows 11

### PR DESCRIPTION
**Pull request by Brouilles**

### CODE REVIEW

File: source/Client/SymEngine.cpp
Comment (gvanem) [inline source/Client/SymEngine.cpp:1]: What about *Windows 11 Server*? 
So if `osvi.wProductType != VER_NT_WORKSTATION`, it is a *Server* I think.

Comment (Brouilles) [reply source/Client/SymEngine.cpp:1]: Windows 11 "server" doesn't exist. Windows Server 2022 is always Windows 10 based.

**Comment by bchavez** [comment]
@gvanem are we okay to merge this? can I get your PR approval?

**Comment by gvanem** [comment]
Sure. Go ahead.

**Pull request merged**